### PR TITLE
Replace BroadcastMessage criteria with filter

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

File: spec/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 1]
```diff
@@ -8,8 +8,14 @@ properties:
...
-  criteria:
-    $ref: "#/components/schemas/Condition"
+  filter:
+    type: string
+    description: >
+      The filter that will be used on customer selection during the broadcast message processing.
+
+      The syntax is the same as the `query` filter, without `filter=` prefix.
+
+      For example: `firstName:John`, `firstName:John;lastName:Doe`, etc.
```
Comment (adamaltman) [inline spec/components/schemas/BroadcastMessages/BroadcastMessage.yaml]: You should give a real example.  OAS allows you to add an example here.

**Comment by vova07** [comment]
please re-review

**Comment by vova07** [comment]
x-tagGroups was updated in another merged PR.

File: spec/components/schemas/BroadcastMessages/BroadcastMessage.yaml (hunk 1 above)
Comment (vova07) [reply spec/components/schemas/BroadcastMessages/BroadcastMessage.yaml]: Done!

**Comment by vova07** [comment]
please re-review

**Pull request merged**